books/free-programming-books-bl.md

```diff
@@ -1,26 +1,119 @@
 ### Index
 
+* [Android](#android)
+* [Bootstrap](#bootstrap)
 * [C](#c)
+* [C++](#c-plus-plus)
 * [Competitive Programming](#competitive-programming)
-* [JavaScript](#JavaScript)
+* [C Sharp](#c-sharp)
+* [CSS](#css)
+* [Dart](#dart)
+* [Git](#git)
+* [HTML](#html)
+* [Java](#java)
+* [JavaScript](#javascript)
+* [jQuery](#jquery)
+* [Kotlin](#kotlin)
 * [Machine Learning](#machine-learning)
+* [NodeJS](#nodejs)
+* [OOP](#oop)
 * [Misc](#misc)
+* [PHP](#php)
 * [Python](#python)
+* [ReactJS](#reactjs)
+* [Svelte](#svelte)
+* [TypeScript](#typescript)
+* [Visual Studio Code](#visual-studio-code)
+
+
+### Android
+
+* [Android Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3p9xzYLFGdfYliIRBLVDRV5) - Anisul Islam
+* [Android Firebase Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oDurEmECb5U_BZ1hrLaHx-) - Anisul Islam
+* [Android SQLite Database Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oJBRutwsFgUKrKJCjv9K3p) - Anisul Islam
+* [Android Tutorials Material Design](https://www.youtube.com/playlist?list=PLgH5QX0i9K3ru-TfN-YsRWKe4EEOLrWjn) - Anisul Islam
+
+
+### Bootstrap
+
+* [Bootstrap 4 Bangla Tutorial](https://www.youtube.com/playlist?list=PL_XxuZqN0xVBr2NqbL3q71nk5FX8zB0nK) - Stack Learner
+* [Bootstrap 5 and 4 Bangla Tutorials](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oC_wmWEZa2xWxJauIRQ9kG) - Anisul Islam
 
 
 ### C
 
+* [C - All you need to know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVASsjyqiNzgjUWHbDkN2Scy) - Stack Learner
+* [C Programming Bangla Tutorial Course](https://www.youtube.com/playlist?list=PLgH5QX0i9K3pCMBZcul1fta6UivHDbXvz) - Anisul Islam
 * [Computer Programming](http://cpbook.subeen.com/p/blog-page.html) - Tamim Shahriar Subeen 
+* [Pattern Printing in C](https://www.youtube.com/playlist?list=PLgH5QX0i9K3oTxQhx2kejYmQn6qtRULCD) - Anisul Islam
+
+
+### C++
+
+* [C++ Bangla Tutorial Course](https://www.youtube.com/playlist?list=PLgH5QX0i9K3q0ZKeXtF--CZ0PdH1sSbYL) - Anisul Islam
 
 
 ### Competitive Programming
 
-* [Dynamic Programming Book](https://dp-bn.github.io) - Tasmeem Reza and Mamnoon Siam 
+* [Dynamic Programming Book](https://dp-bn.github.io) - Tasmeem Reza and Mamnoon Siam
+
+
+### C Sharp
+
+* [C# and ASP.NET MVC Full Bangla Tutorial BITM](https://www.youtube.com/playlist?list=PL_g-DE60bXDBpjMPUWGbmCLHnQDIIcw-6) - Learn With Nirash
+* [C# bangla tutorial | Basic to advance in depth bangla course](https://www.youtube.com/playlist?list=PLbC4KRSNcMnqQakB2xlZPoaV6uau4wTIt) - Learn Hunter
+* [C# OOP (object oriented programming) BanglaTutorials](https://www.youtube.com/playlist?list=PLqCbg_KAOnCe1RLKP2SVmSHZOCD-fWe3p) - Asp Dot Net Explorer
+
+
+### CSS
+
+* [CSS3 and CSS4 Weird Parts Bangla Tutorial](https://www.youtube.com/playlist?list=PL_XxuZqN0xVD3oeT3ckKBmnc7krm-SZl2) - Stack Learner
+* [CSS Complete Course in Bangla 2021](https://www.youtube.com/playlist?list=PLgH5QX0i9K3qjCBXjTmv7Xeh8MDUUVJDO) - Anisul Islam
```

Comment:
CSS before CSS3

Reply:
![Screenshot (886)](https://user-images.githubusercontent.com/64195132/105522672-b9e6d880-5d07-11eb-83ec-d18c8b4b955d.png)

I did it now.